docs/topics/rn-resolved-issues.adoc

```diff
@@ -17,19 +17,23 @@ At the time of the release, the following resolved issues have been identified a
 |Component
 |Summary
 
-|link:https://issues.redhat.com/browse/WINDUP-3201[WINDUP-3201]
-|Red Hat CodeReady Studio extension
-|The plugin cannot be used in Red Hat CodeReady Studio 12.21.0.GA.
+|link:https://issues.redhat.com/browse/WINDUP-2622[WINDUP-2622]
+|Web console on OpenShift Container Platform
+|Custom rules do not work on OpenShift Container Platform.
 
-|link:https://issues.redhat.com/browse/WINDUP-3125[WINDUP-3125]
-|ItelliJ IDEA plugin and Visual Studio Code extension
-|The *Apply QuickFix* menu option is not disabled after a QuickFix is applied.
+|link:https://issues.redhat.com/browse/WINDUP-3234[WINDUP-3234]
+|Rules engine
+|When an analysis using source mode is run on an application source directory that has the target subdirectory, the contents of the target subdirectory are added to the analysis scope. This leads to duplication or possible errors in the results.
 
-|link:https://issues.redhat.com/browse/WINDUP-3134[WINDUP-3134]
-|ItelliJ IDEA plugin
-|After you run an analysis, the *Mark as Complete* menu option is not displayed.
+|link:https://issues.redhat.com/browse/WINDUP-3263[WINDUP-3263]
+|Migration - Application
+|The `excludePackages` argument does not work.
 
-|link:https://issues.redhat.com/browse/WINDUP-3136[WINDUP-3136]
-|Visual Studio Code extension
-|If you run an analysis with an incorrect _JAVA_HOME_ environmental variable, the analysis continues to run after you correct the variable and the *Run* option is disabled.
+|link:https://issues.redhat.com/browse/WINDUP-3233[WINDUP-3233]
+|Rules engine
+|POM files inside a binary are not consistently analyzed.
+
+|link:https://issues.redhat.com/browse/WINDUP-3336[WINDUP-3336]
+|Web UI & Windup as a Service (`windup-web`)
+|Package discovery problem in the MTA web console when running in source mode.
 |====
```

Comment:
@RichardHoch 
Use {ProductShortName} instead of MTA.  And {WebName} instead of 'web console'. And, I think, we are not supposed to use 'UI' - should be 'web console' instead, that is, {WebName}.

Reply:
Since {WebName} = web console; I wrote out "Web console" where the upper case W is needed.